src/interfaces/stepper-interfaces/AddPeopleAndRole/org-person-interface.ts

```diff
@@ -11,5 +11,6 @@ export interface OrgPersonIF {
     orgName: string
   },
   roles: Roles[];
-  address: BaseAddressObjIF;
```

Comment:
Is `BaseAddressObjIF` now obsolete?

Reply:
Not quite, it's still used in the IncorporationAddressIF applied on the OfficeAddresses component.